src/test/shared/utilities/editorUtilities.test.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import * as path from 'path'
+import * as semver from 'semver'
+import { closeAllEditors, assertTabCount, createTestWorkspaceFolder, openATextEditorWithText } from '../../testUtil'
+import { getOpenFilesInWindow } from '../../../shared/utilities/editorUtilities'
+
+// VSCode tab APIs are available since 1.68.0
+function shouldRunTheTest(): boolean {
+    return (semver.valid(vscode.version) && semver.gte(vscode.version, '1.68.0')) as boolean
```

Comment:
before this line, we can check `getMinVscodeVersion()` and throw if it is >= 1.68. That will let us know we can remove `shouldRunTheTest()`.

https://github.com/aws/aws-toolkit-vscode/blob/bfd220fb09749ef345fb94c58fabb668ae9fa5b3/src/shared/vscode/env.ts#L95

Reply:
@justinmk3 sorry didn't follow this one, did you mean remove this func and repalce it with 
```
semver.gte(getMinVsCodeVersion(), '1.68.0')
```
?

Reply:
No. I mean just add a check like:

```
function shouldRunTheTest(): boolean {
  if (semver.gte(getMinVsCodeVersion(), '1.68.0')) {
    throw Error('this function is no longer needed, remove it')
  }
  return (semver.valid(vscode.version) && semver.gte(vscode.version, '1.68.0'))
```

Reply:
updated